internal/pkg/aws/ec2/ec2.go

```diff
@@ -59,24 +83,49 @@ func New(s *session.Session) *EC2 {
 	}
 }
 
+// ListVPC returns IDs of all VPCs.
+func (c *EC2) ListVPC() ([]string, error) {
+	var vpcs []*ec2.Vpc
+	response, err := c.client.DescribeVpcs(&ec2.DescribeVpcsInput{})
+	if err != nil {
+		return nil, fmt.Errorf("describe VPCs: %w", err)
+	}
+	vpcs = append(vpcs, response.Vpcs...)
+
+	for response.NextToken != nil {
+		response, err = c.client.DescribeVpcs(&ec2.DescribeVpcsInput{
+			NextToken: response.NextToken,
+		})
+		if err != nil {
+			return nil, fmt.Errorf("describe VPCs: %w", err)
+		}
+		vpcs = append(vpcs, response.Vpcs...)
+	}
+	var vpcNames []string
+	for _, vpc := range vpcs {
+		vpcNames = append(vpcNames, aws.StringValue(vpc.VpcId))
+	}
+
+	return vpcNames, nil
+}
+
 // ListVPCSubnets lists all subnets given a VPC ID.
-func (c *EC2) ListVPCSubnets(vpcID string) (*Subnets, error) {
+func (c *EC2) ListVPCSubnets(vpcID string, opts ...ListVPCSubnetsOpts) ([]string, error) {
```

Comment:
Same advice regarding the "name" tag - so that customers can see the names if they exist.